core/client/src/main/java/alluxio/client/file/options/MountOptions.java

```diff
@@ -44,6 +45,7 @@ public static MountOptions defaults() {
   private MountOptions() {
     mReadOnly = false;
     mProperties = new HashMap<>();
+    mShared = false;
```

Comment:
I think by default it should be conservative, that the mounted point is not shared publicly. This prevents accidental data leak. User should explicitly enabled `shared` if they really want to.